### ISSUE TITLE
Make it more explicit on the front page/Readme what pubs is/isnot wants/doesnt want to be compared with other similar projects (papis, bibsearch, etc)

### ISSUE THREAD

**Issue by m040601** [pubs/pubs#204]
First of all thank you for your work on this project.

Although I'm not a developer I am a command line fan and academic power user.
And as others I have been eagerly testing and comparing a lot of  similar apps in this area. 

All these very usefull (and niche) cli doc/bib/ref management tools like papis, pubs etc , really deserve that their developers cooperate (or at least know that the others exist and what the others are doing). And at least pay attention to the "heavy weights" like Zotero et al. After all, end users already have a choice and a system in place, so you'll will only be willing to try new stuff if it adds something new.

Because I can never tell 100% from the readme's what the app does/does not, the first thing I do is installed them all and compare. This also helps me make a better choice for what are really my needs.

Of course not all these projects share "exactly the same" scope, ambitions , goals but there is a large intersection, and "market need" catering for all those command line users in this area of bib/ref/docs/pdfs management. Whether Power users, academic users, or simple users.There is ALWAYS something missing in one, that the other might have already figured out.

Even if the pubs developers plan to "stick to their own path" and dont have any special interest in cooperating with others, I would like to request and suggest the following.

Make it easier for potential first time users bumping first time into this project.
Make it more prominent on the Readme or the front page, and/or
 to a quick overview section:

- Why should I install pubs ? (If I already use XYZ ?)
- Make it more explicit what pubs is and is not COMPARED with others.
- What does pubs do (doesnt/will never do/wants to do ) compared with similar  ? Pubs ? bibsearch ? etc ...
- What will I gain from pubs if I already use XYZ (Zotero, etc ...)

**Comment by benureau** [comment]
Aaaaahh... yes. The thing is, that's not really what we want to do (@omangin might have a different opinion though). To give you background, we started pubs to answer a personal need, in our scientific work, to have an efficient, scriptable tool to manage the articles we read. And we decided to share it because other people might have the same need as ours. But we did not do it to create a popular tool. If a lot of people were to take interest in pubs, that would mechanically mean more work for us (issues, features requests, PR), which would take us away from our work. Since we created pubs to increase our scientific efficiency, this is not the goal. So we're not really promoting pubs in any way, or really willing to try to convince you in the readme that you should use pubs. 

As far as the other tools, since we use pubs, we're not using them day to day (or, for most of them, at all), and we would not actually know how we compare. If we were selling pubs, you would have those nice tables with green checkmarks and red crosses with columns for our competition, but this is not actually what we're doing here. We're not trying to stake a slice of the market. And beside, we don't really want to define pubs through comparisons. We're just (and so far) committed to maintain and improve pubs to better serves its function: manage scientific articles through the command line. 

To be fair, pubs does 80-90% of what I wants it to do. There are still some amazing features we could add, but again, when deciding between working on my actual scientific endeavors and pubs, pubs rarely wins (usually once a month).

There's one thing where you are right: the readme needs a refresh, we added quite a bit of features recently and discovering them is hard. I have to fix that. I'll probably do that with the next release.

Thanks for your thoughtful feedback.

**Comment by m040601** [comment]
Thanks for  the time you took to write down this perfect answer.
This is exactly what I was needing, and does  clarify my ideas about what to expect from pubs.
Hope  I can provide you later, if you have interest, with more  feedback from my own experience and personal usage of pubs that may be relevant for the pubs  project.


> There's one thing where you are right: the readme needs a refresh,

just copy/paste bits from this same answer to the front page. I'm sure it will help others like me bumping first time into pubs

**Comment by benureau** [comment]
The idea of a community space on the wiki is a great idea indeed. I'll put things in motion this weekend. Thanks for the suggestion.

**Comment by zamlz** [comment]
I agree, this was very well worded answer. Pubs is great in that it's
simple and modular in a Unix sort of way. And to that end, I think it does
it's job perfectly.

**Comment by benureau** [comment]
@m040601, @zamlz, and everyone else: we wish for the wiki to be a community space. It is available for anyone to add content, share tips and discuss usage and workflow. It's pretty much a blank page right now, so feel free to shape it anyway you want. Anyone should possibly be able to edit it, do tell me if you have any issue.
